data/src/agenttemplates/agent.go

```diff
@@ -0,0 +1,57 @@
+// Merlin is a post-exploitation command and control framework.
```

Comment:
We will want to move this from the `data/src/` directory to something in the `pkg` directory. I intend for the `data/src` directory to be for "The source code of 3rd party tools". All compiled code should be in the `pkg` or cmd` directory.

Reply:
Agree - fair point. I guess it will fit better in the `cmd` directory, will fix in the next commit